README.md

```diff
@@ -263,20 +263,17 @@ These values are used to calculate metrics. For example, the speaking pace may s
 Query table "usermodel" to get a list of speakers and some recording statistics on them.
 
 ## Modify recorder uuid
-The browser used to record your phrases persists the user uuid and user_name in it's local storage.
-
-> Values in browsers local storage, file system structure and sqlite database should always match!
+The browser used to record your phrases persists the user uuid and user_name in it's localStorage to keep it synchron with sqlite and filesystem.
```

Comment:
Small one - looks like "synchron" got cut off.
Optional - if you surround terms like `uuid` and `user_name` with backticks eg ` it helps highlight them and indicate that they are code / variable names.